lib/__tests__/findUndefinedIdentifiers-test.js

```diff
@@ -178,6 +178,22 @@ it('knows about objects', () => {
   ).toEqual(new Set(['foo', 'uuid']));
 });
 
+it('knows about default parameters', () => {
+  expect(
+    findUndefinedIdentifiers(
+      parse(
+        `
+    foo({
+      bar(a = 123) {},
+    });
+
+    function x(b = 456) {}
+  `,
+      ),
+    ),
+  ).toEqual(new Set(['foo']));
```

Comment:
Sorry for the weird formatting used in this file, and thanks for rolling with it. I believe it's a result of running `prettier` on the code.